MovieApp/MovieApp/MovieCell.swift

```diff
@@ -0,0 +1,34 @@
+import UIKit
+
+class MovieCell: UICollectionViewCell {
+    
+    static let cellIdentifier = "cellid"
```

Comment:
We use the class name for the reuse identifiers. That get's useful when you have multiple TVs or CVs in the app and multiple cells.
String gives you convenience init which allows this:
```swift
static let reuseIdentifier = String(describing: MovieCell.self)
```